code/game/machinery/vending/vendor_types/squad_prep/squad_tl.dm

```diff
@@ -89,7 +90,6 @@ GLOBAL_LIST_INIT(cm_vending_gear_tl, list(
 GLOBAL_LIST_INIT(cm_vending_clothing_tl, list(
 		list("STANDARD EQUIPMENT (TAKE ALL)", 0, null, null, null),
 		list("Standard Marine Apparel", 0, list(/obj/item/clothing/under/marine, /obj/item/clothing/shoes/marine/knife, /obj/item/clothing/gloves/marine, /obj/item/device/radio/headset/almayer/marine, /obj/item/clothing/head/helmet/marine/rto), MARINE_CAN_BUY_UNIFORM, VENDOR_ITEM_MANDATORY),
-		list("M4 Pattern Armor", 0, /obj/item/clothing/suit/storage/marine/medium/rto, MARINE_CAN_BUY_ARMOR, VENDOR_ITEM_MANDATORY),
```

Comment:
Don't make the FTL have to pay for his set of M4, also add the light, medium, heavy options if you're going to keep this in.

Reply:
good point i will add the option for him to get the other type of armor like the rifleman.
concerning the M4 i am waiting for an maintainer opinion.
personally i don't think FTL should have a free more powerful armor as it could attract people just for that.